src/box/memtx_engine.cc

```diff
@@ -1412,6 +1413,9 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
 	memtx->state = MEMTX_INITIALIZED;
 	memtx->max_tuple_size = MAX_TUPLE_SIZE;
 	memtx->force_recovery = force_recovery;
+	if (sort_threads == 0)
+		sort_threads = get_nprocs();
+	memtx->sort_threads = sort_threads;
```

Comment:
Suggest default behavior `sort_threads = max(1, (int)get_nprocs() - 1)`

Try not to step on the same rake - do not consume all cores by default.

Reply:
I guess it won't much help as threads number limit is added to keep others Tarantool instances running while this one is starting. Other instances number can be more then one and also Tarantool have more then one thread that can be loaded with work. I guess it is better to use introduced option to limit starting Tarantool according to the installation needs.

Yet I'll add low limit 1 (and high too) on sorting threads.